data/persons.txt

```diff
@@ -788,7 +788,7 @@ ship "Ursa Polaris"
 	plural "Ursa Polaris"
 	sprite "ship/localworldship"
 	attributes
-		category "Heavy Warship"
+		category "Unclassified"
```

Comment:
```suggestion
		category "Utility"
```

Reply:
I do agree with this though - the Ursa Polaris isn't the same thing as Becca's asteroid - it is still a ship with a constructed body, even if that's partly with an asteroid. It makes sense it either keeps "Heavy Warship, as it is a ship that battles, or "Utility", if we are considering an Asteroid being a base. It's still a "-ship" either way", so I wouldn't say it's "unclassified".